x-pack/plugins/logstash/public/plugin.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup, CoreStart, Plugin } from 'src/core/public';
+import { first, map } from 'rxjs/operators';
+import {
+  HomePublicPluginSetup,
+  FeatureCatalogueCategory,
+} from '../../../../src/plugins/home/public';
+import { LicensingPluginSetup } from '../../licensing/public';
+import { ManagementSetup } from '../../../../src/plugins/management/public';
+
+// @ts-ignore
+import { PLUGIN } from '../common/constants';
+import {
+  LogstashLicenseService,
+  // @ts-ignore
+} from './services';
+// @ts-ignore
+import { registerEditSection } from './sections/pipeline_edit';
+// @ts-ignore
+import { registerListSection } from './sections/pipeline_list';
+import { SecurityPluginSetup } from '../../security/public';
+
+interface SetupDeps {
+  home?: HomePublicPluginSetup;
+  licensing: LicensingPluginSetup;
+  management: ManagementSetup;
+  security?: SecurityPluginSetup;
+}
+
+export class LogstashPlugin implements Plugin<void, void, SetupDeps> {
+  public setup(core: CoreSetup, plugins: SetupDeps) {
+    const logstashLicense$ = plugins.licensing.license$.pipe(
+      map(license => new LogstashLicenseService(license))
+    );
+    const section = plugins.management.sections.getSection('logstash')!;
```

Comment:
@mattkime Do you know where this section is defined? I wasn't able to track it down. Should it be moved into this plugin?

Reply:
Its defined here - `src/plugins/management/public/management_service.ts:83`

I do think it would make sense to move it to this plugin.

Reply:
Got it, thank you.